barcode/barcode_utils.py

```diff
@@ -2,8 +2,8 @@
 
 import base64
 import struct
-import skip32
-import code128
+import barcode.skip32
+import barcode.code128
 import os
```

Comment:
Fair enough.  In general this is probably the first time we've had a dependency for a plugin which the core ``uber`` plugin doesn't already list in its ``requirements.txt``.  To my knowledge all of the other plugin ``requirements.txt`` files are empty.

I'm guessing that the reason why you included these here is because puppet and magbot don't know to pull dependencies from other plugins' ``requirements.txt`` files.  That's definitely worth fixing in the long run, but also definitely something we can ignore for now in favor of a quick kludge this close to MagClassic.